checkstyle/import-control.xml

```diff
@@ -341,7 +341,9 @@
   <subpackage name="coordinator">
     <subpackage name="group">
       <allow pkg="org.apache.kafka.common.message" />
+      <allow pkg="org.apache.kafka.common.protocol" />
```

Comment:
For my curiosity, are these lines needed in the context of this pull request or they are a remnant from a previous change?

Reply:
Yes. The generated code needs this.